tests/integration/suite/daprd/tracing/serviceinvocation/serviceinvocation.go

```diff
@@ -57,6 +58,11 @@ func (i *invoke) Setup(t *testing.T) []framework.Option {
 		} else {
 			i.traceparent.Store(false)
 		}
+		if baggage := r.Header.Get("baggage"); baggage != "" {
```

Comment:
Please do not pollute existing tests, and create new ones

Reply:
This was more about being thorough rather than polluting the tests. This is a similar vein for my comment to you in the output binding test. Its apart of tracing, and is testing middleware functionality. This test is focused on tracing as a whole for service invocation, baggage is part of tracing and therefore makes sense to be apart of this test and is not polluting it.

Reply:
Please create different test (files) for each scenario so that a single test is focused on a single scenario. This makes it much clearer what area it is testing, a clear signal as to what is broken when a failure occurs, and help to preserve what the initial test was written for in the first place.